int32 x_coord
int32 y_coord
float32 conf
